src/beautifiers/eslint.coffee

```diff
@@ -23,12 +24,25 @@ module.exports = class ESLintFixer extends Beautifier
       allowUnsafeNewFunction ->
         importPath = Path.join(projectPath, 'node_modules', 'eslint')
         try
-          CLIEngine = require(importPath).CLIEngine
+          context.which('eslint').then((globalEslintPath) ->
+            if globalEslintPath != 'eslint'
+              importPath = Path.resolve(globalEslintPath, '../../lib/node_modules/eslint')
```

Comment:
The usage of `context.which('eslint')` and the `../../lib/node_modules/eslint` concerns me.

I would prefer something more formal and less fragile.

[`requireg`](https://www.npmjs.com/package/requireg) can be used to require global modules.

See https://github.com/Glavin001/atom-beautify/pull/2009/files#r171870409 for what I'd like to see:

```coffeescript
try
  eslint = require(importPath) # Local ESLint for project
catch
  try
    eslint = requireg('eslint') # Globally installed ESLint
  catch
    eslint = require('eslint') # ESLint packaged with Atom-Beautify
```

Reply:
That is really a good way, I will try to change the method and test if that is okay on my mac os environment.

Reply:
This won't work. requireg only searches for a global install if `require(...)` fails.

See [requireg priority](https://www.npmjs.com/package/requireg#resolution-priority)

Reply:
Instead of `requireg` looks like `import-global` should work: https://github.com/sindresorhus/import-global